BadHabbit/BadHabbit/UserStories/SingUpScreen/SignUpScreenRouter.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SignUpScreenRouter.swift
+//  BadHabbit
+//
+//  Created by Viktoria Misiulia on 24/07/2023.
+//
+
+import Foundation
+
+protocol SignUpRouterProtocol {
+    func presentSignInAppleScreen()
+}
+
+final class SignUpScreenRouter: SignUpRouterProtocol {
+    
+    let nextScreen: StoryAssembly
+    let globalCoordinator: IGlobalCoordinator
+    
+    init(nextScreen: StoryAssembly,
+         globalCoordinator: IGlobalCoordinator) {
+        self.nextScreen = nextScreen
+        self.globalCoordinator = globalCoordinator
+     }
+
+    func presentSignInAppleScreen() {
+        let vc = nextScreen.assembleStory()
```

Comment:
we're presenting singIn screen, so rename story assembly to signIn in init parameter and class variable

Reply:
Renamed to SignInDataAssembly (as class was named in pr #10)